libraries/lib-numeric-formats/formatters/ParsedNumericConverterFormatter.cpp

```diff
@@ -580,6 +581,403 @@ class ParsedNumericConverterFormatter final : public NumericConverterFormatter
+ *  needed to create that format output. This is used for the pop-up
+ *  list of formats to choose from in the control.          */
+static BuiltinFormatString TimeConverterFormats_[] =  {
+   {
+   /* i18n-hint: Name of time display format that shows time in seconds */
+   { XO("seconds") },
+   /* i18n-hint: Format string for displaying time in seconds. Change the comma
+    * in the middle to the 1000s separator for your locale, and the 'seconds'
+    * on the end to the word for seconds. Don't change the numbers. */
+   XO("01000,01000 seconds")
+   },
+
+   {
+   /* i18n-hint: Name of time display format that shows time in seconds
+    * and milliseconds (1/1000 second) */
+   { XO("seconds + milliseconds") },
+   /* i18n-hint: Format string for displaying time in seconds and milliseconds 
+    * as fractional seconds. Change the comma in the middle to the 1000s separator
+    * for your locale, and the 'seconds' on the end to the word for seconds.
+    * Don't change the numbers. The decimal separator is specified using '<' if
+    * your languages uses a ',' or to '>' if your language uses a '.'. */
+     { XO("01000,01000>01000 seconds"),
+       XO("milliseconds") }
+   },
+
+   {
+   /* i18n-hint: Name of time display format that shows time in hours, minutes
+    * and seconds */
+   { XO("hh:mm:ss") },
+   /* i18n-hint: Format string for displaying time in hours, minutes and
+    * seconds. Change the 'h' to the abbreviation for hours, 'm' to the
+    * abbreviation for minutes and 's' to the abbreviation for seconds. Don't
+    * change the numbers unless there aren't 60 seconds in a minute in your
+    * locale */
+   XO("0100 h 060 m 060 s")
+   },
+
+   {
+   /* i18n-hint: Name of time display format that shows time in days, hours,
+    * minutes and seconds */
+   { XO("dd:hh:mm:ss") },
+   /* i18n-hint: Format string for displaying time in days, hours, minutes and
+    * seconds. Change the 'days' to the word for days, 'h' to the abbreviation
+    * for hours, 'm' to the abbreviation for minutes and 's' to the
+    * abbreviation for seconds. Don't change the numbers unless there aren't
+    * 24 hours in a day in your locale */
+   XO("0100 days 024 h 060 m 060 s")
+   },
+
+   {
+   /* i18n-hint: Name of time display format that shows time in hours,
+    * minutes, seconds and hundredths of a second (1/100 second) */
+   { XO("hh:mm:ss + hundredths") },
+   /* i18n-hint: Format string for displaying time in hours, minutes, seconds
+    * and hundredths of a second. Change the 'h' to the abbreviation for hours,
+    * 'm' to the abbreviation for minutes and 's' to the abbreviation for seconds
+    * (the hundredths are shown as decimal seconds). Don't change the numbers
+    * unless there aren't 60 minutes in an hour in your locale.
+    * The decimal separator is specified using '<' if your language uses a ',' or
+    * to '>' if your language uses a '.'. */
+     { XO("0100 h 060 m 060>0100 s"),
+       XO("centiseconds") }
```

Comment:
The "hack" that the commit comment mentions is, I now see, the `GetFraction()` function.  Please mention that specifically in the commit comment.

Reply:
I also notice now that this big commit in 2021 171d8cf added the special hack of '>' in the mini-language, but the long comment explaining the syntax of the language wasn't update for it.  And there is a confusing long i18n comment repeated in many places.

I haven't thought whether there was a better way to localize the decimal point.  Have you?

Reply:
We need to use system one. However this is a problem for future us :-)

Reply:
The long comment all out the mini language might still be updated